wbooks/src/interfaces/bookState.ts

```diff
@@ -2,6 +2,4 @@ import { Book } from '@interfaces/books';
 
 export interface BookState {
   books: Book[];
-  error: string;
-  loading: boolean;
```

Comment:
Are you sure? What happens with the `completeState` in the reducer?

Reply:
You're right, I'm now considering `booksLoading` and `booksError`